tests/Unit/Evolution/Systems/RelativisticEuler/Valencia/BoundaryCorrections/Test_Rusanov.cpp

```diff
@@ -0,0 +1,154 @@
+#include <string>
+
+#include "Evolution/Systems/RelativisticEuler/Valencia/BoundaryCorrections/Factory.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/BoundaryCorrections/Rusanov.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/System.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Helpers/Evolution/DiscontinuousGalerkin/BoundaryCorrections.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace {
+struct ConvertPolytropic {
+  using unpacked_container = bool;
+  using packed_container = EquationsOfState::PolytropicFluid<true>;
+  using packed_type = bool;
+
+  static inline unpacked_container unpack(
+      const packed_container& /*packed*/,
+      const size_t /*grid_point_index*/) noexcept {
+    return true;
+  }
+
+  [[noreturn]] static inline void pack(
+      const gsl::not_null<packed_container*> /*packed*/,
+      const unpacked_container& /*unpacked*/,
+      const size_t /*grid_point_index*/) {
+    ERROR("Should not be converting an EOS from an unpacked to a packed type");
+  }
+
+  static inline size_t get_size(const packed_container& /*packed*/) noexcept {
+    return 1;
+  }
+};
+
+struct ConvertIdeal {
+  using unpacked_container = bool;
+  using packed_container = EquationsOfState::IdealFluid<true>;
+  using packed_type = bool;
+
+  static inline unpacked_container unpack(
+      const packed_container& /*packed*/,
+      const size_t /*grid_point_index*/) noexcept {
+    return false;
+  }
+
+  [[noreturn]] static inline void pack(
+      const gsl::not_null<packed_container*> /*packed*/,
+      const unpacked_container& /*unpacked*/,
+      const size_t /*grid_point_index*/) {
+    ERROR("Should not be converting an EOS from an unpacked to a packed type");
+  }
+
+  static inline size_t get_size(const packed_container& /*packed*/) noexcept {
+    return 1;
+  }
+};
+
+namespace helpers = TestHelpers::evolution::dg;
+
+template <size_t Dim, typename EosType>
+void test(const size_t num_pts, const EosType& equation_of_state) {
+  tuples::TaggedTuple<hydro::Tags::EquationOfState<EosType>> volume_data{
+      equation_of_state};
+  tuples::TaggedTuple<
+      helpers::Tags::Range<hydro::Tags::RestMassDensity<DataVector>>,
+      helpers::Tags::Range<hydro::Tags::SpecificInternalEnergy<DataVector>>,
+      helpers::Tags::Range<hydro::Tags::SpecificEnthalpy<DataVector>>,
+      helpers::Tags::Range<hydro::Tags::SpatialVelocity<DataVector, Dim>>>
+      ranges{std::array<double, 2>{{1e-6, 1.0}},
```

Comment:
I'm not sure what the lower range should be. `1.0e-30` still seems reasonable to test.

Also `1e-6` -> `1.0e-6`